13output.tex

```diff
@@ -75,16 +75,16 @@
 
 \hypertarget{VirginUnfished}{}
 \subsubsection[Virgin Spawning Biomass vs. Unfished Spawning Biomass]{\protect\hyperlink{VirginUnfished}{Virgin Spawning Biomass vs. Unfished Spawning Biomass}}
-Unfished is the condition for which reference points (benchmark) are calculated. Virgin Spawning Biomass (B0, Bzero) is the initial condition on which the start of the time series depends. If biology or spawner-recruitment parameters are time-varying, then the benchmark year input in the forecast file tells the model which years to average in order to calculate ``unfished''. In this case, virgin recruitment and/or the virgin spawning biomass will differ from their unfished counterparts. Virgin recruitment and spawning biomass are reported in the mgmt\_quant portion of the sd\_report and are now labeled as ``unfished'' for clarity. Note that if ln(R0) is time-varying, then this will cause unfished to differ from virgin. However, if regime shift parameter is time-varying, then unfished will remain the same as virgin because the regime shift is treated as a temporary offset from virgin. Virgin spawning biomass is denoted as SPB\_virgin and spawning biomass unfished is denoted as SPB\_unf in the report file.
+Unfished is the condition for which reference points (benchmark) are calculated. Virgin Spawning Biomass ($B_{0}$, also seen as Bzero in the Report file) is the initial condition on which the start of the time series depends. If biology or spawner-recruitment parameters are time-varying, then the benchmark year input in the forecast file tells the model which years to average in order to calculate ``unfished''. In this case, virgin recruitment and/or the virgin spawning biomass will differ from their unfished counterparts. Virgin recruitment and spawning biomass are reported in the mgmt\_quant portion of the sd\_report and are now labeled as ``unfished'' for clarity. Note that if $ln(R_{0})$ is time-varying, then this will cause unfished to differ from virgin. However, if regime shift parameter is time-varying, then unfished will remain the same as virgin because the regime shift is treated as a temporary offset from virgin. Virgin spawning biomass is denoted as SSB\_virgin and spawning biomass unfished is denoted as SSB\_unfished in the report file.
```

Comment:
I would delete the abbreviation for virgin spawning biomass in the second sentence.  I think it is unneeded and the final sentence tells the user how it is labeled in the Report file which is the critical piece of information.